stubs/tifftools/constants.pyi

```diff
@@ -0,0 +1,47 @@
+from typing import Any, Generator, Generic, TypeVar, overload
+
+_ConstantAttr = Any
```

Comment:
What is this doing? Just aliasing `Any`?

Reply:
Yeah, I was trying use this alias to conceptually separate the type for `TiffConstant` attribute values, indicating all the related places that it's used. I can add a code comment if you think this is unclear.

Reply:
I changed the name, added an explicit `TypeAlias` type, and added a comment.